.eslintrc.yml

```diff
@@ -6,6 +6,7 @@ ignorePatterns:
 rules:
   '@typescript-eslint/strict-boolean-expressions': off
   '@typescript-eslint/restrict-template-expressions': off
+  '@typescript-eslint/naming-convention': off
```

Comment:
This guy is always kicking off about `event_name` etc. Bored of it to be honest.